oops/surface/ringplane.py

```diff
@@ -62,20 +59,41 @@ def __init__(self, origin, frame, radii=None, gravity=None,
         self.origin    = Path.as_waypoint(origin)
         self.frame     = Frame.as_wayframe(frame)
         self.gravity   = gravity
-        self.elevation = elevation
+        self.elevation = float(elevation)
         self.modes     = modes
         self.nmodes    = len(self.modes)
-        self.epoch     = epoch
+        self.epoch     = float(epoch)
 
         if radii is None:
             self.radii = None
         else:
             self.radii    = np.asfarray(radii)
             self.radii_sq = self.radii**2
 
+        # Save the unmasked version of this surface
+        if radii is None:
+            self.unmasked = self
+        else:
+            self.unmasked = RingPlane(self.origin, self.frame,
+                                      radii = None,
```

Comment:
When I have a long list of keyword=value pairs in a function call, such that it makes sense to align each item vertically with one per line, I find this _much_ more readable. I believe this is compliant with PEP8 under bullet 1 of https://peps.python.org/pep-0008/#a-foolish-consistency-is-the-hobgoblin-of-little-minds. If you really disagree, I will change it.

Reply:
I won't force a change, but I think of it differently. When I see "x = a" I know it's an assignment statement. When I see "x=a" I know it's a function argument. To me when they're on separate lines like this is actually when I want the spacing to be gone even more, because it's a visual aid to know we're still within the argument list.

Reply:
Of course, these actually _are_ assignments. The only real difference is that they are assignments to variables in the namespace of the function being called, not the namespace of the function doing the calling. I will say more in Slack.